tests/bson/bson-fromPHP_error-002.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+BSON\fromPHP(): Unknown BSON type class
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+use MongoDB\BSON as BSON;
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+class UnknownType implements BSON\Type {}
+
+echo throws(function() {
```

Comment:
Correct. I wasn't sure if we wanted to do that (before your email response and spec change). I'll add a second test for it, though.